acceptance/setup/pre_suite/80_add_dev_repo.rb

```diff
@@ -1,6 +1,7 @@
 install_opts = options.merge( { :dev_builds_repos => ["PC1"] })
 repo_config_dir = 'tmp/repo_configs'
-if (test_config[:install_type] == :package and not test_config[:skip_presuite_provisioning])
+if (test_config[:install_type] == :package and test_config[:package_build_version] and not test_config[:skip_presuite_provisioning])
```

Comment:
Might wrap this guard since if nothing else, it "goes of the side" of the gh diff window.